### ISSUE TITLE
negative samples

### ISSUE THREAD

**Issue by Mayzy** [AlexeyAB/darknet#4630]
Hello, does yolov3 need to add unlabeled pictures (negative samples) to the dataset? What is the basis for doing this? Which paper do you refer to？

**Comment by AlexeyAB** [comment]
Most neural networks (and Yolo too) do not require negative samples.
But most neural networks  (and Yolo too) will improve accuracy by using negative samples.

Basis:
1.  more training data - higher accuracy (it’s easier for you to collect negative examples than positive ones, as you don't need to label objects)

2. very often in custom datasets all objects are located on the same background, so you need to explain to the network that this particular background is not an object. For 1-stage detectors (Yolo, SSD, ...) this is much more important than for multi-stage detectors (faster-RCNN, ...), because receptive field of the final activation  is much larger than the detection bbox, so it sees the whole images,  so you need to explain to the network that background isn't an object.

I do not know what paper describes it.
